packages/errors/package.json

```diff
@@ -2,18 +2,23 @@
   "name": "@packages/errors",
   "version": "0.0.0-development",
   "description": "Error definitions and utilities for Cypress",
-  "main": "index.js",
-  "browser": "src/index.ts",
+  "main": "cjs/index.js",
+  "module": "esm/index.js",
+  "browser": "esm/index.js",
+  "files": [
+    "cjs/*",
+    "esm/*"
+  ],
+  "types": "src/index.ts",
   "scripts": {
-    "build": "../../scripts/run-if-ci.sh tsc || echo 'type errors'",
```

Comment:
this script is actually not used anywhere else except this package so I will delete it before merging in